inc/servervars.php

```diff
@@ -83,7 +83,7 @@ function get_major_version($version) {
   // Uses official API for version checking, but is not optimized for
   // version-checking against multiple release tiers.
   function get_keymanweb_version($tier) {
-    $json = @file_get_contents(KeymanHosts::Instance()->r_keymanweb_com . "/code/get-version/web/$tier");
+    $json = @file_get_contents(KeymanHosts::Instance()->api_keyman_com . "/version/web/$tier");
```

Comment:
Another separate fix which just crept in. No big deal, r.keymanweb.com was just calling out to api.keyman.com anyway, but should help performance!

Reply:
Uh... might want to nix the comments in line 83 and 84 then.  I left that (and made the comment in #20) as an example of the 'right' way to do things.

I'm pretty sure that `get_keymanweb_version($tier)` isn't actually used right now.

Reply:
I think the comment is still valid for the function, isn't it? It is the official API for version checking (more official even than r.keymanweb.com which is deprecated), isn't it?